Running "gmake test" was failing for tests 58 and 59 because the wrong
test name was specified.

This patch should be sent upstream.

--- popt-1.19/tests/testit.sh.orig	2024-07-24 14:41:39.308462826 +0200
+++ popt-1.19/tests/testit.sh	2024-07-24 14:43:41.072037091 +0200
@@ -111,8 +111,8 @@
 run test1 "test1 - 57" "arg1: 0 arg2: (none) aBits: foo,baz" --bits foo,bar,baz,!bar
 
 run test1 "test1 - 58" "\
-Usage: test1 [-I?] [-c|--cb2=STRING] [--arg1] [-2|--arg2=ARG]
-        [-3|--arg3=ANARG] [-onedash] [--optional=STRING] [--val]
+Usage: lt-test1 [-I?] [-c|--cb2=STRING] [--arg1] [-2|--arg2=ARG]
+	      [-3|--arg3=ANARG] [-onedash] [--optional=STRING] [--val]
         [-i|--int=INT] [-s|--short=SHORT] [-l|--long=LONG]
         [-L|--longlong=LONGLONG] [-f|--float=FLOAT] [-d|--double=DOUBLE]
         [--randint=INT] [--randshort=SHORT] [--randlong=LONG]
@@ -120,7 +120,7 @@
         [-x|--bitxor] [--nstr=STRING] [--lstr=STRING] [-I|--inc]
         [-c|--cb=STRING] [--longopt] [-?|--help] [--usage] [--simple=ARG]" --usage
 run test1 "test1 - 59" "\
-Usage: test1 [OPTION...]
+Usage: lt-test1 [OPTION...]
       --arg1                      First argument with a really long
                                   description. After all, we have to test
                                   argument help wrapping somehow, right?
